src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/WalletNotDeletedException.java

```diff
@@ -0,0 +1,8 @@
+package pl.byczazagroda.trackexpensesappbackend.exception;
+
+public class WalletNotDeletedException extends AppRuntimeException{
+
+    public WalletNotDeletedException() {
+        super("Value does not exist in the database, please change your request");
```

Comment:
Let's return message:
  "_The wallet has not been deleted_"
User does not need to know about database existence or about "changing request" - whatever that mean for him.
Name of exception is "Wallet not deleted" - That doesn't mean id of the wallet was wrong. That doesn't mean something bad occur with database. That even doesn't mean database is used in this project. **That mean wallet is not deleted**.

Reply:
I understand, the message was changed.